tasks/appraisal.rake

```diff
@@ -166,11 +166,11 @@ TRACER_VERSIONS = %w[
   3.0
   3.1
   3.2
-  jruby-9.2.8.0
```

Comment:
Maybe we should convert this `%w[` to an actual array, so that we can add the comment inline, e.g:

```
3.1
3.2
#jruby-9.2.8.0 # TODO: jruby-9.2.8.0 disabled for possible removal
jruby-9.3
jruby-9.4
```

Minor sidenote: I'm not quite a fan of this Rubocop rule of forcing people to use `%w[]` % the likes. As an old Perl user I get where it comes from and there are definitely legit uses where it really adds to readability, but I feel like most of the time it doesn't really add to readability, or forces inconsistency and breaks rhythm which hampers readability.

The above is an example of that, where the jruby-9.2.8.0 specific comment gets to be hoisted out because of Rubocop.

Reply:
Yeah, I really dislike this Rubocop rule as well. I can never remember what's the correct one to use...

Fixed in 25534a2425a70c78d4aeee507bcf47196e25fdb6 .

Reply:
Don't like `%w[]` +1

Reply:
Thanks for the feedback about the `%w[]` y'all. I've jotted a note to open a separate PR to tweak our Rubocop rules about this :)